### PR TITLE
Add Command-line usage of Application

### PR DESCRIPTION
**Pull request by PIYUSHgoyal16**

### CODE REVIEW

File: _documentation/05-User-Manual.md [hunk 1]
```diff
@@ -201,3 +201,125 @@ Apart from configuring printing defaults for the printer, these are the other pr
...
+
+        <application-name> shutdown
+
+### Devices
+
+* **Adding Device**
+    <br> Add a printer queue.
+
+        <application-name> add -d <DEVICE_NAME> -m <DRIVER-NAME> -v <DEVICE-URI> [-o <NAME> = <VALUE>]
+
```
Comment (tillkamppeter) [inline _documentation/05-User-Manual.md:280]: Tell the user here that the he can obtain the possible '<DRIVER NAME>' values via the call with the "drivers" subcommand and the possible '<DEVICE-URI>' values via the call with "devices" subcommand.

File: _documentation/05-User-Manual.md [hunk 2]
```diff
@@ -201,3 +201,125 @@ Apart from configuring printing defaults for the printer, these are the other pr
...
 
     This option helps in testing printer functionality, by issuing a print command for a sample test page. 
+
+## Working with Command-Line
+
+Before start working with your device, you must know about the capabilities of your printer/scanner by viewing the set of sub-commands and options supported by your printer/scanner. For retrieving this information, you need to pass the `--help` argument:
+
+    <application-name> --help
+
+Although, each application may have a different usage-callback function, the default list of sub-commands and options available with all applications is shown below.
```
Comment (tillkamppeter) [inline _documentation/05-User-Manual.md]: If this is a user manual you should not mention the "usage-callback" function as this is an internal functionality of the Printer Application which only a developer sees, not a user. Simply write that some Printer Applications can have additional sub-commands.

File: _documentation/05-User-Manual.md [hunk 3]
```diff
@@ -287,30 +326,43 @@ Although, each application may have a different usage-callback function, the def
 
             <application-name> cancel -j <JOB-ID>
 
+___
+
 ### Other Utilities 
 
 * **List devices**
 
+    List connected printers.
```
Comment (tillkamppeter) [inline _documentation/05-User-Manual.md:335]: List **the** connected printers.

File: _documentation/05-User-Manual.md [hunk 4]
```diff
@@ -287,30 +326,43 @@ Although, each application may have a different usage-callback function, the def
...
 
 * **List jobs**
 
+    List pending print jobs.
+
         <application-name> jobs
 
 * **List printer options**
 
+    List supported options.
```
Comment (tillkamppeter) [inline _documentation/05-User-Manual.md:353]: List **the** supported options.

File: _documentation/05-User-Manual.md (hunk 1 above)
Comment (PIYUSHgoyal16) [reply _documentation/05-User-Manual.md:290]: Added, thanks for the feedback :-)

File: _documentation/05-User-Manual.md (hunk 2 above)
Comment (PIYUSHgoyal16) [reply _documentation/05-User-Manual.md]: Thanks, Changed the content.

**Review by tillkamppeter — APPROVED** [review]

**Comment by tillkamppeter** [comment]
@PIYUSHgoyal16, thanks for the fixes. Can you please resolve the conflicts so that this PR can get merged? Thanks.

**Comment by PIYUSHgoyal16** [comment]
@tillkamppeter there were quite a few conflicts. As a result, I have opened a new Pull Request: https://github.com/OpenPrinting/openprinting.github.io/pull/117

Apologies for the inconvenience.

**Pull request closed without merging**